src/types/onyx/RecentWaypoints.ts

```diff
@@ -7,6 +7,6 @@ type RecentWaypoints = {
 
     /** The longitude of the waypoint */
     lng: number;
-};
+} | null;
```

Comment:
Why this?

Reply:
Because of this [here](https://github.com/Expensify/App/blob/a365583e8f79f6172ada155ca3b774d84b8f6416/src/libs/actions/Transaction.js#L122C13-L122C13) we are passing null

Reply:
fixed @thiagobrez @fabioh8010